velox/functions/prestosql/types/TimestampWithTimeZoneType.h

```diff
@@ -125,4 +124,18 @@ class TimestampWithTimeZoneTypeFactories : public CustomTypeFactories {
 
 void registerTimestampWithTimeZoneType();
 
+static constexpr int32_t TIME_ZONE_MASK = 0xFFF;
+static constexpr int32_t MILLIS_SHIFT = 12;
+
+inline int64_t unpackMillisUtc(int64_t dateTimeWithTimeZone) {
```

Comment:
Would be great to add some unit tests for pack/unpack in TimestampWithTimeZoneTypeTest.cpp.

Reply:
Already added, thanks.